invenio_app_ils/ui/backoffice/src/pages/ItemList/ItemTable/ItemTable.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Table } from 'semantic-ui-react';
+
+class ItemTable extends Component {
+  navToDetails(itemId) {
+    this.props.history.push(`/backoffice/items/${itemId}`);
+  }
+
+  renderData(items) {
+    return items.map(item => {
+      let meta = item.metadata;
+      return (
+        <Table.Row
+          key={meta.item_pid}
+          onClick={() => this.navToDetails(meta.item_pid)}
+        >
+          <Table.Cell collapsing>{meta.barcode}</Table.Cell>
+          <Table.Cell collapsing>{meta.status}</Table.Cell>
+          <Table.Cell collapsing>{meta.item_pid}</Table.Cell>
+        </Table.Row>
+      );
+    });
+  }
+
+  render() {
+    let { data } = this.props;
+    let items = data.hits.hits;
+    return (
+      <Table color={'black'} selectable celled>
```

Comment:
We should put in common this colors in a theme css files somewhere defining these global constants.

Reply:
I couldn't agree more, in one of my previous commits, I had put a `variables.scss` for general overrides of semantic. It looks is not that simple. Semantic is written in `less` and we are using `scss`. There is a library called `semantic-ui-scss`, which we can have a look at, but either way overriding is not supported out of the box for our case.